tests/services/kernels/test_api.py

```diff
@@ -136,6 +138,18 @@ def client_fetch(*parts, headers={}, params={}, **kwargs):
     assert restarted_kernel['id'] == kernel2['id']
     assert restarted_kernel['name'] == kernel2['name']
 
+    # Start a kernel with a path
+    r = await fetch(
+        'api', 'kernels',
+        method='POST',
+                body=json.dumps({
+            'name': NATIVE_KERNEL_NAME,
```

Comment:
Use `'pyimport/kernel'` for the NATIVE_KERNEL_NAME.

Reply:
Thx. This change makes the test green (or at least yellow) on my latptop.